central/detection/lifecycle/manager_impl_test.go

```diff
@@ -79,6 +83,8 @@ func makeIndicator() (*storage.ProcessBaselineKey, *storage.ProcessIndicator) {
 		DeploymentId:  uuid.NewV4().String(),
 		ContainerName: uuid.NewV4().String(),
 		PodId:         uuid.NewV4().String(),
+		ClusterId:     uuid.NewV4().String(),
+		Namespace:     uuid.NewV4().String(),
```

Comment:
Now that the creation of the baseline has been separated from sending it, functions such as `KeyToID` are actually being called, instead of being mocked out. That means that the indicator here has to be complete.

Reply:
This is now outdated.